pubsub/google/cloud/pubsub_v1/subscriber/client.py

```diff
@@ -49,6 +53,23 @@ class in order to define your own consumer. This is primarily
             arguments.
     """
     def __init__(self, policy_class=thread.Policy, **kwargs):
+        # Use a custom channel.
+        # We need this in order to set appropriate default message size and
+        # keepalive options.
+        target = '{host}:{port}'.format(
+            host=subscriber_client.SubscriberClient.SERVICE_ADDRESS,
+            port=subscriber_client.SubscriberClient.DEFAULT_SERVICE_PORT,
+        )
+        kwargs.setdefault('channel', grpc_helpers.create_channel(
+            target=target,
+            scopes=subscriber_client.SubscriberClient._ALL_SCOPES,
+            options={
+                'grpc.max_send_message_length': -1,
+                'grpc.max_receive_message_length': -1,
+                'grpc.keepalive_time_ms': 30000,
+            }.items(),
```

Comment:
This (using `.items()`) is a strange indirection, just use a list/tuple or pairs?